README.md

```diff
@@ -1,10 +1,12 @@
-README
-
-you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
-You can compile now the program by writing "jam" in the terminal from inside the source-directory.
-
-You need at least r1ac3 rc   to be able to run the application without major problems.
-
-The source code is very dirty. It needs to be cleaned and restructured and some things reimplemented. I did most of the work late at night and under time pressure I decided to implement the solution that was the fastest to implement. I hope this summer i will have some time to clean it
-
-.
\ No newline at end of file
+README
+
+![Puri screenshot](Screenshot.png "Puri")
+
+you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
+You can compile now the program by writing "jam" in the terminal from inside the source-directory.
+
+You need at least r1ac3 rc   to be able to run the application without major problems.
+
+The source code is very dirty. It needs to be cleaned and restructured and some things reimplemented. I did most of the work 
+late at night and under time pressure I decided to implement the solution that was the fastest to implement. I hope this 
+summer i will have some time to clean it
```

Comment:
Apparently, he didn't have time that summer, many years ago... :)
Let's remove that whole paragraph, too.

Reply:
yup.  it will live on it the commit history, doesn't need to show up in the readme though. ;)